csi/util.go

```diff
@@ -369,6 +369,24 @@ func ensureMountPoint(path string, mounter mount.Interface) (bool, error) {
 	return isMnt, err
 }
 
+// ensureBindMountPoint evaluates whether a path is a valid mount point for bind mount.
+// In case the path does not exist, it will create a regular file for bind mount.
+// In case where the mount point exists, the mount point will be cleared and replaced by a new regular file.
+func ensureBindMountPoint(path string, mounter mount.Interface) error {
+	logrus.Infof("Trying to ensure bind mount point %v", path)
```

Comment:
```suggestion
func recreateBindMountPoint(path string, mounter mount.Interface) error {
	logrus.Infof("Recreating bind mount point %v", path)
```
Also, update the function comment, since it does more than just evaluation.